### ISSUE TITLE
Bulk add permissions error

### ISSUE THREAD

**Issue by laralfield** [bwdinc/simmer#42]
When trying to add bulk ingredients (I tried a few variations on this list just to be sure it wasn't a rogue character or something setting off a bug), message display is "No don't have permission to do this." 

![screen shot 2015-04-14 at 4 01 12 pm](https://cloud.githubusercontent.com/assets/1518034/7146368/0ee37a86-e2c1-11e4-8276-f627d050bebf.png)

**Comment by ChaseWiseman** [comment]
Is this happening on multiple different recipes or is it isolated? Are you able to add bulk instructions?

**Comment by laralfield** [comment]
It seems that it was only happening on that recipe, and just the time I submitted the ticket. I just successfully added the list below to a new recipe, and another old recipe (just to test), plus the original blueberry recipe, and each time, both bulk ingredients and bulk instructions were successfully added. 

3/4 cup (175 ml) pure maple syrup
1/3 cup (80 ml) heavy cream
1 1/2 teaspoons apple cider vinegar
A couple pinches of sea salt
6 tablespoons (3 ounces or 85 grams) unsalted butter, softened
2 tablespoons (25 grams) granulated sugar
1 large egg
1/2 teaspoon vanilla extract
1 cup (125 grams) all-purpose flour
1 teaspoon baking powder
1/4 teaspoon fine sea or table salt
To serve: crème fraîche and more sea salt, to taste

Feel free to close this one, may have just been a random occurance :) Thanks for checking.
